go.mod

```diff
@@ -3,8 +3,7 @@ module github.com/integr8ly/integreatly-operator
 go 1.13
 
 require (
-	github.com/3scale/3scale-operator v0.6.0
-	// github.com/3scale/3scale-operator v0.2.1-0.20200730110533-c3b57b704d73
+	github.com/3scale/3scale-operator v0.2.1-0.20201120164439-360b1f382716
```

Comment:
@trepel should this one be the 0.7.0 tag when it's available?

Reply:
@laurafitzgerald good catch. It seems so, yes. I will be checking the availability of the tag and update once it appears.